src/Circlereader/util/optionsAttribute.ts

```diff
@@ -0,0 +1,631 @@
+    });
+    attributes.push({attribute: 'keep_num_dims', value: fullyConnectedOpt.keepNumDims()});
+    attributes.push({
+      attribute: 'asymmetric_quantize_inputs',
+      value: fullyConnectedOpt.asymmetricQuantizeInputs()
+    });
+  }
+
+  static getSoftMaxAttr(operator: Operator, attributes: Array<NodeAttributes>) {
+    let softMaxOpt = new SoftmaxOptions();
+    softMaxOpt = operator.builtinOptions<flatbuffers.Table>(softMaxOpt);
+
+    attributes.push({attribute: 'beta', value: softMaxOpt.beta()});
+  }
+
+  static getConcatenationAttr(operator: Operator, attributes: Array<NodeAttributes>) {
+    let concatenationOpt = new ConcatenationOptions();
+    concatenationOpt = operator.builtinOptions<flatbuffers.Table>(concatenationOpt);
+
+    attributes.push({attribute: 'axis', value: concatenationOpt.axis()});
+    attributes.push({
+      attribute: 'fused_activation_function',
+      value: ActivationFunctionType[concatenationOpt.fusedActivationFunction()]
+    });
+  }
+
+  static getAddAttr(operator: Operator, attributes: Array<NodeAttributes>) {
+    let addOpt = new AddOptions();
+    addOpt = operator.builtinOptions<flatbuffers.Table>(addOpt);
+
+    attributes.push({
+      attribute: 'fused_activation_function',
+      value: ActivationFunctionType[addOpt.fusedActivationFunction()]
+    });
+  }
+
+  static getL2NormAttr(operator: Operator, attributes: Array<NodeAttributes>) {
+    let l2NormOpt = new L2NormOptions();
+    l2NormOpt = operator.builtinOptions<flatbuffers.Table>(l2NormOpt);
+
+    attributes.push({
+      attribute: 'fused_activation_function',
+      value: ActivationFunctionType[l2NormOpt.fusedActivationFunction()]
+    });
+  }
+
+  static getLocalResponseNormalizationAttr(operator: Operator, attributes: Array<NodeAttributes>) {
+    let localResponseNomalizationOpt = new LocalResponseNormalizationOptions();
+    localResponseNomalizationOpt =
+        operator.builtinOptions<flatbuffers.Table>(localResponseNomalizationOpt);
+
+    attributes.push({attribute: 'radius', value: localResponseNomalizationOpt.radius()});
+    attributes.push({attribute: 'bias', value: localResponseNomalizationOpt.bias()});
+    attributes.push({attribute: 'alpha', value: localResponseNomalizationOpt.alpha()});
+    attributes.push({attribute: 'beta', value: localResponseNomalizationOpt.beta()});
+  }
+
+  static getLSTMAttr(operator: Operator, attributes: Array<NodeAttributes>) {
+    let lstmOpt = new LSTMOptions();
+    lstmOpt = operator.builtinOptions<flatbuffers.Table>(lstmOpt);
+
+    attributes.push({
+      attribute: 'fused_activation_function',
+      value: ActivationFunctionType[lstmOpt.fusedActivationFunction()]
+    });
+    attributes.push({attribute: 'cell_clip', value: lstmOpt.cellClip()});
+    attributes.push({attribute: 'proj_clip', value: lstmOpt.projClip()});
+    attributes.push({attribute: 'kernel_type', value: LSTMKernelType[lstmOpt.projClip()]});
```

Comment:
To keep your convention...

```suggestion
    attributes.push({attribute: 'lstm_kernel_type', value: LSTMKernelType[lstmOpt.projClip()]});
```